### ISSUE TITLE
[IDEA] add the GNOME shell sound menu extenssion as a default part of atom shell

### ISSUE THREAD

**Issue by sergioad** [ozonos/bug-tracker-BETA#8]
Hello all the OzonOS team, I have an important idea for you and it is: Why do you not add the GNOME shell sound menu extenssion by default on Atom shell and OzonOS? this will make more easier control your music from the main menu and also add more functionallity to it

**Comment by UriHerrera** [comment]
@me4oslav ?

I personally can say that: There's no intention no include everything under the sun. There's a default, which is what we've chosen. Anything else is up to the users own preferences.

**Comment by sergioad** [comment]
@UriHerrera i understand but this is a standard to play music with confort

**Comment by UriHerrera** [comment]
I don't know how truly convenient is to have the controls in the tray vs simply re-opening the player window. It's not like the application is going to load everything again. Even more so since pretty much every keyboard made the past decade has dedicated multimedia keys.

**Comment by sergioad** [comment]
I see :smile:

**Comment by Foggalong** [comment]
I'd be up for including this. I didn't realise GNOME didn't have this feature - pretty much every other DE going ships some kind of feature like this. Does this mean that without it you can't close the music play and have it keep running?

**Comment by sergioad** [comment]
Indeed, for example with Noise an Cinnamon, CInnamon does not supports noise and because that you can not stop the reproduction, you need to re-open the app, stop the reproduction and close

**Comment by theenduser** [comment]
I'm all for including this by default. Sometimes you want to keep an eye on the current window while controlling your music. The extension is very helpful in this regard.

**Comment by sergioad** [comment]
True story :smile:
